src/backends/ncnn/ncnninputconns.h

```diff
@@ -113,6 +113,20 @@ namespace dd
               _in.substract_mean_normalize(_mean.data(), _std.data());
             }
         }
+      else if (_scale != 1.0)
+        {
+          float norm = 1.0 / _scale;
+          if (_bw)
+            {
+              std::vector<float> vscale = { norm };
+              _in.substract_mean_normalize(0, vscale.data());
+            }
+          else
+            {
+              std::vector<float> vscale = { norm, norm, norm };
+              _in.substract_mean_normalize(0, vscale.data());
+            }
+        }
```

Comment:
In torch backend, scale is applied before std and mean. Should this be the preferred behavior ? I feel like if the user provides both scale and mean / std at the same time, and they are not supposed to, an exception should be thrown.

Reply:
Yes, raising the exception should go into another PR I believe.